DigitalLearningSolutions.Data.Tests/Models/CourseSettingsTests.cs

```diff
@@ -190,5 +190,34 @@ public void CourseSettings_should_have_default_ConsolidationExercise_when_wrong_
             // Then
             courseSettings.ConsolidationExercise.Should().BeNull();
         }
+        [Test]
+        public void CourseSettings_should_take_SupportingInformation_from_json()
+        {
+            // When
+            var courseSettings = new CourseSettings("{\"lm:si\":\"Supporting information\"}");
+
+            // Then
+            courseSettings.SupportingInformation.Should().Be("Supporting information");
+        }
+
+        [Test]
+        public void CourseSettings_should_have_default_SupportingInformation_when_not_in_json()

```

Comment:
One thing I hadn't considered before, for string values in the course setting we should probably not use them if they're empty 🙂

Reply:
Yeah neither had I! Fixed in 72ce2a9 🙂